drivers/entropy/entropy_ll_nrfxlib.c

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
There already is RNG driver in the Zephyr. Also, it was optimized in order to met BLE stack requirements.

Reply:
They need all access to the RNG peripheral to go through the closed-source driver in the NRFXLIB .a file.

Reply:
What is all access? The driver is already there.
If it does not meet requirements of the proprietary BLE stack, we should improve it instead of creating new one. Especially when the new one is almost identical to the previous one.

Reply:
This is the same discussion as https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/687#discussion_r278570181, so let's close this discussion and keep it in the other thread.